### ISSUE TITLE
UAT feedback on performance

### ISSUE THREAD

**Issue by AnaBelgun** [TerriaJS/neii-viewer#55]
TBA by the BOM team

**Comment by RacingTadpole** [comment]
Hi Chantelle, what did you want me to do here?

**Comment by ChantelleDoan** [comment]
Hi Arthur,
Matt reported a rendering issue via email (refer below).  I also tested.  I was able to view the same layers ok in Chrome with no rendering issues.  However did experience similar rendering issues (hanging) when using in IE11, especially when trying to view multiple layers (ie select all 3 layers below by using the Shift key).  

Hi Anna
Received the following advice below from one of my team members that these WMS services are all operational. 
Suggest review and reuse of the Geofabric map layer URL's in the new staging NEII Viewer site, thought there does appear to be a rendering issue as shown below. 
http://staging.neiiviewer.nicta.com.au/
 
![geofabric rendering](https://cloud.githubusercontent.com/assets/17890967/20585620/a9261d6e-b250-11e6-9702-95e024cc3352.jpg)

Let us know if you require anything further clarification.

Matt.

**Comment by RacingTadpole** [comment]
This sort of thing can happen in 3D occasionally if the curvature of the Earth means that layers are at different discrete scales - not sure if that's the whole problem here though.
It would be helpful to know if this also happens in 2D?

**Comment by ChantelleDoan** [comment]
I tested in 2D and encounter similar rendering issues as above when using 3D.

Ian Muirhead (Solar) and Jo Sullivan (WDO) conducted testing and also encountered rendering issues.  Ian's test feedback on 2/12/16:
A little feedback, as much of the layout etc is probably locked down
1)	I added annual GHI climatology and then one minute solar NEMSR & the system froze
a.	IE 11.0.96
b.	Repeatable (& not really acceptable in this day and age). Also froze with just NEMSR
2)	In Chrome, Firefox and IE, the links to metadata definitions in the pop-up box you get when clicking on a one minute solar NEMSR dot on the map were quite intermittent on whether it worked or not on first click.  eg: Click "Add Data" button -> select add item for NEII Data Services - Conformant - Air - Annual climatology of daily exposure - Global Horizontal Exposure.  The pop sometime closes but not layer has been added to the map.  Repeat this step the second time and the selected layer is added successfully.

**Comment by RacingTadpole** [comment]
@ChantelleDoan - I'm having trouble reproducing it in 2D - Could you try getting it into such a state in 2D and then hitting "Share", and pasting that share URL into this issue? Thanks.

**Comment by ChantelleDoan** [comment]
Hi Arthur,

Most of the issues rendering issues are identified in IE.  I tried just now to replicate but the screen hangs so I am not able to share the url.  However below is a screenshot of when it hangs.  I selected the SATRN layer -> select the add data button -> screen hangs here.

I have encountered the screen freezing in IE11 often while testing today.

[cid:image001.png@01D24F14.7208B840]
Regards,
Chantelle Doan | NEII Programme Manager
[Description: Description: cid:389443803@21082013-35A7]
Environmental Information Services
Bureau of Meteorology
GPO Box 1289 Melbourne VIC 3001
Level 9, 700 Collins Street, Docklands VIC 3008
Tel: +61 3 9669 4475 | chantelle.doan@bom.gov.au<mailto:chantelle.doan@bom.gov.au> | www.bom.gov.au<http://www.bom.gov.au/>
[app email badge.jpg]<http://www.bom.gov.au/app>

**Comment by RacingTadpole** [comment]
thanks - unfortunately the screenshot didn't make into github - could you paste it directly into https://github.com/TerriaJS/neii-viewer/issues/55 ?

**Comment by ChantelleDoan** [comment]
![freeze](https://cloud.githubusercontent.com/assets/17890967/20947196/39d8bc16-bc62-11e6-92e2-f861285ae11f.jpg)

**Comment by RacingTadpole** [comment]
Great news - we have solved the problem on IE11! It turns out it was a known bug that only affects IE11 on Windows 7, reported here: https://connect.microsoft.com/IE/feedback/details/796745/mouse-events-are-not-delivered-at-all-anymore-when-inside-an-svg-a-use-is-removed-from-the-dom. Unfortunately Microsoft is no longer fixing non-security-related bugs on Win7 IE11. But fortunately there is a work-around that we've now included in https://github.com/TerriaJS/terriajs/pull/2287.
